changelog.d/20930_socket_sink_accepts_metrics.fix.md

```diff
@@ -0,0 +1,3 @@
+Socket sink can now accept metric events (when using `native_json` codec).
```

Comment:
```suggestion
Socket sink can now accept metric events when using codecs that support decoding metrics.
```

I think the full list is actually `native_json`, `native`, and, recently added, `influxdb`. I'd just leave that off though.

Reply:
wouldn't it be "when using codecs that supports *encoding* metrics" instead?

Reply:
Thanks. Applied your suggestion @jorgehermo9

Reply:
Doh, yes, _encoding_. Thanks for the catch @jorgehermo9